calendar.html

```diff
@@ -78,7 +77,7 @@
 
 			  <div id="footer">
 			  	<hr>
-			  	Copyright &copy; 2017 SCS Concordia
+			  	Copyright &copy; 2018 SCS Concordia
```

Comment:
This is why dynamic Django templates were nice haha.

Reply:
yessss ;-;